src/App.js

```diff
@@ -1,29 +1,12 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import React from 'react';
+import Calculator from './components/calculator';
+// eslint-disable-next-line react/prefer-stateless-function
+class App extends React.Component {
+  render() {
+    return (
+      <Calculator />
+    );
+  }
```

Comment:
- [ ] it is always a good practice to make sure you fix all linter errors instead of disabling them, Linter is a tool for identifying and reporting errors when you use standard rules. 💯 😉 

- [ ] You have also done the same thing in the `Calculator` component. Kindly fix them.
- **Hint**: I guess you are trying to make a way through the class-based component. I will advise you to go with React Pure component. [ReadMore](https://www.geeksforgeeks.org/reactjs-pure-components/)

eg: 
```.js
export default class App extends React.PureComponent{
   render(){
      return <h1>blablabla</h1>;
   }
}
```

In that example, App is still a class-based component

Reply:
hi @BenMukebo I fixed it for App component

Reply:
Congratulations @PinkMoon25  🥳 🎉 💪🏻